### ISSUE TITLE
Nightblade (2handed sword) not proccing)

### ISSUE THREAD

**Issue by Volkyte** [chromiecraft/chromiecraft#1190]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

Nightblade item (2handed sword) noit proccing at all after a lot of time fighting. proc rate seems VERY low (like 1% or so)

### Expected Blizzlike Behaviour

https://wowwiki-archive.fandom.com/wiki/Chance_on_hit
https://wowwiki-archive.fandom.com/wiki/Proc

### Source

https://wowwiki-archive.fandom.com/wiki/Proc

### Steps to reproduce the problem

1. equip nightblade
2. 2.attack

### Extra Notes

https://wowgaming.altervista.org/aowow/?item=1982

https://imgur.com/a/4ZDkwjD

**Comment by Eldarix** [comment]
After about 5 minutes, it happened strangely 3 times in a row (AC HASH/COMMIT: 05bbf4a) :

![WoWScrnShot_071421_180907](https://user-images.githubusercontent.com/82579073/125646460-7040da02-8c8e-4b9b-b221-1ff9831a174d.jpg)

**Comment by Azcobu** [comment]
Thanks for the report. I've checked Nightblade in the DB and found it currently has a Proc Per Minute rating of 0.8. It is also correctly set up to have a chance to be triggered on hit. Just to make sure, I then spent 10 minutes whacking a target dummy with it and scored 7 procs - although 3 of them came in one 10 second window. But there was also about a 4 minute window where I didn't get any procs at all. That's RNG for you I guess.

![WoWScrnShot_071521_002046](https://user-images.githubusercontent.com/81782124/125644749-1400ecf5-847d-4af3-8d45-1c8c30955bc6.jpg)

As to the PPM value, I haven't found any independent verification for the 0.8 value. TBC Wowhead has the usual vague and unhelpful comments - "seems like the proc is really bad; maybe 1 proc per 5 mobs or so" is about it, although there's another from vanilla claiming a 1/6 rate, which I doubt was ever true. But for now it looks as if this is working as intended, so pending any further evidence that this is incorrect, will close this ticket.
